README.md

```diff
@@ -2,7 +2,7 @@
 
 **"abcxyz Common Packages" is not an official Google product.**
 
-<img src="./pkg.png" alt="Logo" width="100" />
+<img src="./pkg.png" alt="Logo" width="300" />
```

Comment:
I can give you the svg if you want - it'll be smaller and auto-scale.

Reply:
I am not worried about it unless you are....😁